roles/edpm_extra_mounts/defaults/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+# Copyright 2024 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+# All variables intended for modification should be placed in this file.
+
+# All variables within this role should have a prefix of "edpm_extra_mounts"
+edpm_extra_mounts_debug: "{{ (ansible_verbosity | int) >= 2 | bool }}"
+edpm_extra_mounts_hide_sensitive_logs: true
+
+edpm_extra_mounts: []
+edpm_default_mounts: [
+  {name: 'hugepages1G', path: '/dev/hugepages/hugepages1G', opts: 'pagesize=1G', fstype: 'hugetlbfs'},
+  {name: 'hugepages2M', path: /dev/hugepages/hugepages2M, opts: 'pagesize=2M', fstype: 'hugetlbfs'}]
```

Comment:
oh this is not what i meant 

/dev/hugepages/hugepages1G should be 

/dev/hugepages1G and 
/dev/hugepages2M

this is whyyoru having permission issues 

/dev/hugepages is not a normal directory
its a hugetlbfs mount point

Reply:
done